### DISCUSSION TITLE
Configuring Management Interface Externally (AWS)

### DISCUSSION THREAD

**Discussion by derekseisollc** [Security-Onion-Solutions/securityonion#6050] [Unsupported Versions] [answered]
So I am having an issue where I have SO in AWS and have done the entire setup. All security groups are configured to allow my external IP inbound to the AWS external IP. However, when I go to connect to the SO Web UI it establishes the initial connection but then fails because it attempts to force the private IP of the interface to communicate as opposed to the external IP. Is there a way within the setup or with a command that I can allow the Web UI to run on both the internal IP and External?

**Comment by dougburks** [comment, marked as answer]
Setup allows you to specify how you would like to connect to the web interface:
![image](https://user-images.githubusercontent.com/1659467/139663560-32b0d9bd-406d-49a3-9ee2-aefa25bfcc67.png)

You can also change this after running Setup:
https://docs.securityonion.net/en/2.3/url-base.html
